src/main.rs

```diff
@@ -79,23 +81,66 @@ fn get_args(current_dir: &OsStr) -> clap::ArgMatches {
                 .long("no-color")
                 .help("Turns off the colored output"),
         )
-        .arg(
-            Arg::with_name("fix")
-                .short("f")
-                .long("fix")
-                .help("Automatically fixes warnings"),
-        )
-        .arg(
-            Arg::with_name("no-backup")
-                .long("no-backup")
-                .help("Prevents .env files from being backed up when modified by -f/--fix"),
-        )
         .arg(
             Arg::with_name("quiet")
                 .short("q")
                 .long("quiet")
                 .help("Doesn't display additional information"),
         )
+        .subcommand(SubCommand::with_name("checks").about("Shows list of available checks"))
```

Comment:
It seems to me that this can be a little confusing, especially if we still need a subcommand with the name `check`. Maybe it makes sense to rename to something like: `names` or `list ? Or any another suggestions?
By and large, names are used not only for checks 👀

Reply:
I like `list`, thank you! 👍